jib-core/src/main/java/com/google/cloud/tools/jib/filesystem/FileOperations.java

```diff
@@ -36,7 +36,11 @@ public static void copy(ImmutableList<Path> sourceFiles, Path destDir) throws IO
       PathConsumer copyPathConsumer =
           path -> {
             // Creates the same path in the destDir.
-            Path destPath = destDir.resolve(sourceFile.getParent().relativize(path));
+            Path parent = sourceFile.getParent();
+            if (parent == null) {
+              throw new IOException("Parent path unavailable for " + sourceFile);
```

Comment:
Hmm... how about this?
```suggestion
              throw new IllegalArgumentException("Parent path unavailable for " + sourceFile);
```